eax/src/lib.rs

```diff
@@ -148,10 +149,12 @@ pub const P_MAX: u64 = 1 << 36;
 pub const C_MAX: u64 = (1 << 36) + 16;
 
 /// EAX nonces
-pub type Nonce<NonceSize> = GenericArray<u8, NonceSize>;
+pub type Nonce<NonceSize> = Array<u8, NonceSize>;
 
 /// EAX tags
-pub type Tag<TagSize> = GenericArray<u8, TagSize>;
+pub type Tag<TagSize> = Array<u8, TagSize>;
+
+type OutputSize<T> = <T as OutputSizeUser>::OutputSize;
```

Comment:
Is there a type alias already existing for that? I couldn't find one.

Reply:
I don't believe so. Seems like a reasonable idea though if you want to PR it to `crypto-common`.

Reply:
```suggestion
# to drop once https://github.com/RustCrypto/traits/pull/1533 releases.
type OutputSize<T> = <T as OutputSizeUser>::OutputSize;
```